app/src/main/java/io/horizontalsystems/bankwallet/modules/main/MainViewModel.kt

```diff
@@ -70,6 +78,15 @@ class MainViewModel(
         transactionTabEnabledLiveData.postValue(accounts.isNotEmpty())
     }
 
+    private fun showWhatsNew() {
+        viewModelScope.launch{
+            delay(2000)
```

Comment:
Let's move the delay into the if block. Otherwise it will delay even if it doesn't show the changelog.